redaxo/src/addons/be_style/CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 Changelog
 =========
 
+Version 2.9.0 – 22.01.2020
```

Comment:
Changelog bitte nicht hier im PR schreiben.

Gregor fügt das alles in #3177 zusammen.

Reply:
Ok, wollte nur die Änderungen auch dokumentieren und @schuer 's Änderung

Reply:
Ich gehe immer jeden einzelnen Commit seit letztem Release durch, und füge es in die Changelogs ein.
Habe hier bereits begonnen: https://github.com/redaxo/redaxo/pull/3177
Bin aktuell am 4.1. angelangt.

Und da arbeite ich nach festem Schema. Zum Beispiel setze ich für die Beta überall das Datum auf XX.XX.2020, und im Release lasse ich es dann ersetzen.

Deswegen mache ich immer lieber alles in einem Rutsch, weil ich dann weiß, dass mein Schema funktioniert, und nicht zum Beispiel hier plötzlich ein "22.01." auftaucht ;) 

Da du aber für deine Änderungen eine schöne ausführliche Liste hast, und da wir kurz vor Release stehen, lass es ausnahmsweise mal drin.

Bloß diese Änderung ist notwendig:
```suggestion
Version 2.9.0 – XX.XX.2020
```

Reply:
Ok @gharlan 
Ich ändere also jetzt noch in meinem Branch das Datum!?

Reply:
entweder so, oder oben auf "commit suggesion"

Reply:
Also dieses github ist schon echt geil :) !